Automattic/WP-Job-Manager
# Add option to show logo in Recent Jobs widget

Fixes #1367 

#### Changes proposed in this Pull Request:

* Add option in Recent Jobs widget to show logo 

#### Testing instructions:

* Add Recent Jobs Widget to Sidebar
* Check/Uncheck the Show Logo Option


#### Proposed changelog entry for your changes:
* Add option to show logo in Recent Jobs widget
